yarn.lock

```diff
@@ -14156,6 +14164,18 @@ simplebar@^4.0.0:
     resize-observer-polyfill "^1.5.1"
     scrollbarwidth "^0.1.3"
 
+simplebar@^4.2.2:
+  version "4.2.2"
+  resolved "https://registry.yarnpkg.com/simplebar/-/simplebar-4.2.2.tgz#3d8144091274f0daa483179cd54708818f39b2ed"
+  integrity sha512-OWHcnifIbGhB7RBdYS57xsqOUIp9GOKCJw5dOJCuKsV83+1kdKcbIaXaqb6A9zAOIRqhJIdN2qOBrsd4Axe84A==
+  dependencies:
+    can-use-dom "^0.1.0"
+    core-js "^3.0.1"
+    lodash.debounce "^4.0.8"
+    lodash.memoize "^4.1.2"
+    lodash.throttle "^4.1.1"
```

Comment:
those lodash.* deps are out of date and should not be used

Reply:
![image](https://user-images.githubusercontent.com/19857479/65254933-4f170300-dafd-11e9-8d95-db756619ec9a.png)

Reply:
The version resolved of lodash is [4.17.11](https://github.com/Talend/ui/blob/e08f84462b2899c995fb1d49a712d26a4d3e1619/yarn.lock#L9889) so we don't care?
No related dependencies are new here actually so I don't see any problems?

Reply:
lodash.* package do not follow 4.17. They stop at 4.4 so you have lodash twice.

The fact they are not new comes from different reason: they are used in dev deps (so not a bundle pb) + some other components like `reactour ` in fact.

So I just alert on the fact every project will take 50k for a scrollbar.